packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -101,16 +99,21 @@ const Editor = ({
     extensions,
     content,
     onUpdate: ({ editor }) => {
-      const content = editor.getHTML()
-      if (content === '<p></p>') {
+      if (editor.isEmpty) {
         // this is when content of the editor is empty
         // set controller value to empty string instead so the required rule can
         // work properly
         onChange('')
         return
       }
 
-      onChange(content)
+      const content = editor.getHTML()
+      const contentWithBreaks = content.replace(
+        /(<p\s?((style=")([a-zA-Z0-9:;.\s()\-,]*)("))?>)\s*(<\/p>)/g,
```

Comment:
nice catch! this is to handle people who are styling (bolding, underlining) multiple lines right